### PR TITLE
Fix import errors

### PR DESCRIPTION
**Pull request by lassoan**
fixes issues mentioned at https://github.com/pnlbwh/SlicerDiffusionQC/issues/46

### CODE REVIEW

**Comment by jamesobutler** [comment]
@tashrifbillah Could you help with the integration of this PR? Thanks!

**Comment by tashrifbillah** [comment]
Hi all, I am surprised that the PR notification did not make it to my primary inbox. Rest assured, I shall review it shortly.

File: SlicerDiffusionQC/GradQC.py [hunk 1]
```diff
@@ -390,6 +388,8 @@ def observeStatus(caller,_):
 
   def GUI(self):
 
+      from gradqclib.slicerUserInteraction import slicerGUI
```
Comment (tashrifbillah) [inline SlicerDiffusionQC/GradQC.py:391]: Should this be `SlicerDiffusionQC.gradqclib.slicerUserInteraction`? Compared with the above, `gradqclib` directory is not at the same level as that of `diffusionqclib`.

**Comment by tashrifbillah** [comment]
The changes look good to me. If we merge it today, should we be able to see updates in https://slicer.cdash.org/index.php?project=SlicerPreview tomorrow?

I see the tests are failing across all three of Linux, MAC, and WIN. However, the builds are already successful.

**Comment by lassoan** [comment]
> The changes look good to me. If we merge it today, should we be able to see updates in https://slicer.cdash.org/index.php?project=SlicerPreview tomorrow?

Yes, if you merge it before 10pm today then you'll see the new results on the dashboard tomorrow morning.

**Comment by tashrifbillah** [comment]
Hi Andras, the [build](https://slicer.cdash.org/build/2675365) looks okay but we still have one and the most important [test](https://slicer.cdash.org/build/2675365) failing:

```
Traceback (most recent call last):
  File "/work/Preview/S-0-E-b/DiffusionQC/Test/run_test.py", line 7, in <module>
    from plumbum import local
ModuleNotFoundError: No module named 'plumbum'
```

The build log confirms that `plumbum` package was installed but the testing script cannot find it. How should we change the testing infrastructure then:

https://github.com/pnlbwh/SlicerDiffusionQC/blob/master/Test/CMakeLists.txt

and

https://github.com/pnlbwh/SlicerDiffusionQC/blob/cc785145b301c8b8797099ec1527aceb4b90679a/Test/run_test.py#L7

**Pull request merged**